tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/demo/tf-interactive-inference-multi-demo.html

```diff
@@ -0,0 +1,273 @@
+        this.model2 = await tf.loadModel(tf.io.browserHTTPRequest(
+          'data/uci/model.json', {credentials: 'include'}));
+
+        const DATA_PATH = "data/uci/uci_test_examples.json";
+        const testData = d3.json(DATA_PATH).then(data => {
+          data = data.slice(0, 500);
+          this.data = data;
+          for (let i = 0; i < data.length; i++) {
+            this.indicesToInfer[i] = true;
+          }
+          this.$.dash.updateExampleContents_(
+            data.map(item => JSON.stringify(item)), false);
+          this.$.dash.addEventListener('update-example', e => {
+            this.data[e.detail.index] = JSON.parse(e.detail.example);
+            this.indicesToInfer[e.detail.index] = true;
+          });
+          this.$.dash.addEventListener('duplicate-example', e => {
+            this.data.push(
+              JSON.parse(JSON.stringify(this.data[e.detail.index])));
+            this.indicesToInfer[this.data.length - 1] = true;
+          });
+          this.$.dash.addEventListener('delete-example', e => {
+            this.data.splice(e.detail.index, 1);
+            const newIndicesToInfer = {};
+            const oldIndicesToInfer = Object.keys(this.indicesToInfer);
+            for (let i = 0; i < oldIndicesToInfer.length; i++) {
+              const idx = oldIndicesToInfer[i];
+              if (idx < e.detail.index) {
+                newIndicesToInfer[idx] = true;
+              } else {
+                newIndicesToInfer[idx - 1] = true;
+              }
+            }
+            this.indicesToInfer = newIndicesToInfer;
+          });
+          this.$.dash.addEventListener('infer-examples', async e => {
+            const inferences = {};
+            const indices = Object.keys(this.indicesToInfer).sort();
+            inferences.indices = indices;
+            inferences.results = [{classificationResult: {classifications: []}},
+               {classificationResult: {classifications: []}}];
+            const tensors = [];
+            for (let i = 0; i < indices.length; i++) {
+              const idx = +indices[i];
+              tensors.push(this.convertExToTensor(this.data[idx]));
+            }
+            const input = tf.concat(tensors).reshape([tensors.length, 104])
+            const res = this.model.predict(input, {batchSize: 128});
+            const res2 = this.model2.predict(input, {batchSize: 128});
+            const predValues = await res.data();
+            const predValues2 = await res2.data();
+            const PRED_SIZE = 2;
+            for (let i = 0; i < indices.length; i++) {
+              inferences.results[0].classificationResult.classifications[i] =
+                {classes: [{label: '0', score: predValues[i * PRED_SIZE + 1]},
+                           {label: '1', score: predValues[i * PRED_SIZE]}]};
+              inferences.results[1].classificationResult.classifications[i] =
+                {classes: [{label: '0', score: (1-predValues2[i * PRED_SIZE + 1])/2},
```

Comment:
does this cause the 2nd model to often have very different results than the first? if so, might be better just to go with a model1 + [some random small positive or negative value] for model 2

Reply:
removed edits since we now have a real second model